elements/element-stuff-base.html

```diff
@@ -16,13 +16,14 @@
           super.ready();
           this._recomputeStuff();
           this.root.addEventListener('change', this._do.bind(this));
+          this.root.addEventListener('color-picker-selected', this._do.bind(this));
         }
 
         _recomputeStuff() {
           this._stuff = [];
           let els = this.root.querySelectorAll('[name]');
           for (let i = 0 ; i < els.length; i++) {
-            this._stuff.push(els[i].name);
+            this._stuff.push(els[i].getAttribute('name'));
```

Comment:
Why is this change needed?

Reply:
Hi, I'm sorry, I've been really busy lately with my last project :confused:
I was not sure about this change, but it seems that, as the  `name` attribute is not a property of `paper-swatch-picker`, we can't access it directly, that's why I needed to use it as attribute